.github/workflows/ci.yml

```diff
@@ -28,6 +28,6 @@ jobs:
       run: npm run api-extractor
     - name: Run emulator-based integration tests
       run: |
-        npm install -g firebase-tools
+        npm install -g firebase-tools@9.16.0
```

Comment:
The underlying issue has been resolved with latest firebase-tools (v9.16.5). This shouldn't be needed any more.